pkg/trisa/peers/peers.go

```diff
@@ -29,12 +32,25 @@ type Peers struct {
 
 // New creates a new Peers cache to look up peers from context or by endpoint.
 func New(certs *trust.Provider, pool trust.ProviderPool, directoryURL string) *Peers {
-	return &Peers{
+	p := &Peers{
 		certs:        certs,
 		pool:         pool,
 		peers:        make(map[string]*Peer),
 		directoryURL: directoryURL,
 	}
+	if directoryURL == PeersTesting {
+		p.directory = &mock.MockDirectoryClient{}
+		networkPeer := &Peer{
+			parent: p,
+			info: &PeerInfo{
+				CommonName: "test-peer",
+				SigningKey: &rsa.PublicKey{},
+			},
+			client: &mock.MockNetworkClient{},
+		}
+		p.peers["test-peer"] = networkPeer
+	}
```

Comment:
I figured that injecting the mocks like this would work for now since "testing" is not a valid URL, but I understand if we need to find an alternative method that's more decoupled.

Reply:
I see what you mean, and it does follow the pattern of some of what we're doing in GDS with the `TESTING` configuration. However, I think I'd prefer this to be in a `NewMock` function where we can add some documentation, rather than relying on a string - is that ok?